File: week-07/day-2/Workshop 16.5.2023/telephone_book.py
telephone_book = {}

telephone_book['William A. Lathan'] = '405-709-1865'
telephone_book['John K. Miller'] = '402-247-8568'
telephone_book['Hortensia E. Foster'] = '606-481-6467'
telephone_book['Amanda D. Newland'] = '319-243-5613'
telephone_book['Brooke P. Askew'] = '307-687-2982'

print(telephone_book.get('John K. Miller'))

value = {i for i in telephone_book if telephone_book[i] == '307-687-2982'}
print(value)


if telephone_book.get('Chris E. Myers'):
    print('yes')
else:
    print('no')

File: week-02/day-4/Workshop 13.4.2023/src/TelephoneBook.java
import java.util.Arrays;
import java.util.HashMap;
import java.util.Map;

public class TelephoneBook {
    public static void main(String[] args) {
        HashMap <String, String> telephoneBook = new HashMap<>();
        telephoneBook.put("William A. Lathan", "405-709-1865");
        telephoneBook.put("John K. Miller", "402-247-8568");
        telephoneBook.put("Hortensia E. Foster", "606-481-6467");
        telephoneBook.put("Amanda D. Newland", "319-243-5613");
        telephoneBook.put("Brooke P. Askew", "307-687-2982");

        System.out.println(telephoneBook.get("John K. Miller"));

        String number = "307-687-2982";

        for (Map.Entry<String, String> whosNumber : telephoneBook.entrySet()) {
            if (whosNumber.getValue() == number) {
                System.out.println(whosNumber.getKey());
            }
        }


        if (!telephoneBook.containsKey("Chris E. Myers")) {
                System.out.println("no");

        }
    }
}
